### PR TITLE
[#677] Tests for favorite static imports

### PR DESCRIPTION
**Pull request by holgerschill**

### CODE REVIEW

File: org.eclipse.xtend.ide.tests/src/org/eclipse/xtend/ide/tests/contentassist/XImportSectionContentAssistTest.xtend [hunk 1]
```diff
@@ -83,4 +85,106 @@ class XImportSectionContentAssistTest extends AbstractXtendContentAssistBugTest
...
+			
+			class Bar{
+				def void foo(){
+			staticMethod()''')
+	}
+	@Test 
+	def void testStaticFavoriteImports_field(){
+		
+		val jdtPreference = InstanceScope.INSTANCE.getNode(JavaUI.ID_PLUGIN)
+		jdtPreference.put(PreferenceConstants.CODEASSIST_FAVORITE_STATIC_MEMBERS,"org.eclipse.xtend.ide.tests.data.contentassist.StaticClassExample.*")
```
Comment (szarnekow) [inline org.eclipse.xtend.ide.tests/src/org/eclipse/xtend/ide/tests/contentassist/XImportSectionContentAssistTest.xtend]: Can we use StaticClassExample.class.getName()?

File: org.eclipse.xtend.ide.tests.data/src/org/eclipse/xtend/ide/tests/data/contentassist/StaticClassExample.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package org.eclipse.xtend.ide.tests.data.contentassist;
+
+public class StaticClassExample {
+
```
Comment (szarnekow) [inline org.eclipse.xtend.ide.tests.data/src/org/eclipse/xtend/ide/tests/data/contentassist/StaticClassExample.java]: I see blank lines :)

File: org.eclipse.xtend.ide.tests/src/org/eclipse/xtend/ide/tests/contentassist/XImportSectionContentAssistTest.xtend (hunk 1 above)
Comment (holgerschill) [reply org.eclipse.xtend.ide.tests/src/org/eclipse/xtend/ide/tests/contentassist/XImportSectionContentAssistTest.xtend]: Done

File: org.eclipse.xtend.ide.tests.data/src/org/eclipse/xtend/ide/tests/data/contentassist/StaticClassExample.java (hunk 2 above)
Comment (holgerschill) [reply org.eclipse.xtend.ide.tests.data/src/org/eclipse/xtend/ide/tests/data/contentassist/StaticClassExample.java]: Done

**Pull request merged**